Validate and Format there is no space in the end of name or id. (#1572)
demisto/demisto-sdk: CHANGELOG.md

* Fixed an issue where yml files with '!reference' failed to load properly.
* Fixed an issue when `View Integration Documentation` button was added twice during the download and re-upload.
* Fixed an issue when `(Partner Contribution)` was added twice to the display name during the download and re-upload.
* Added to **validate** command a validation that a content item's id and name will not end with spaces.

> * The **format** command will now remove whitespaces at the end of content item's id and name.

Reviewer: ```suggestion
* The **format** command will now remove trailing whitespaces from content items' id and name fields.
```
